File: sdkwork-app-sdk-go/types/faq_detail_vo.go
package types

// FAQ详情信息
type FaqDetailVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id string `json:"id"`
	Question string `json:"question"`
	Answer string `json:"answer"`
	CategoryId string `json:"categoryId"`
	CategoryName string `json:"categoryName"`
	ViewCount int `json:"viewCount"`
	HelpfulCount int `json:"helpfulCount"`
}

File: sdkwork-app-sdk-typescript/src/types/faq-detail-vo.ts
/** FAQ详情信息 */
export interface FaqDetailVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** FAQ ID */
  id?: string;
  /** 问题 */
  question?: string;
  /** 回答 */
  answer?: string;
  /** 分类ID */
  categoryId?: string;
  /** 分类名称 */
  categoryName?: string;
  /** 浏览量 */
  viewCount?: number;
  /** 有帮助数 */
  helpfulCount?: number;
}
